src/lib/math/pcurves/pcurves_impl/pcurves_impl.h

```diff
@@ -1973,7 +1973,7 @@ const auto& SSWU_C1()
 * See RFC 9380 ("Hashing to Elliptic Curves") section 6.6.2
 */
 template <typename C>
-inline auto map_to_curve_sswu(const typename C::FieldElement& u) -> typename C::AffinePoint {
+inline auto map_to_curve_sswu(const typename C::FieldElement& u) -> C::AffinePoint {
```

Comment:
I'm surprised that the typename in the parameter can't go.

Reply:
Yeah I don't understand the problem here either